mise.toml

```diff
@@ -0,0 +1,2 @@
+[tools]
+node = "22"
```

Comment:
Can you share what is this file for? 🙏

Reply:
Upsa.

Reply:
[7092de6](https://github.com/commercetools/nodejs/pull/1930/commits/7092de6ed2dd3464e14ea64604ba4b1d51828787)